build_java.pl

```diff
@@ -138,60 +148,59 @@ sub setup_vars {
     grab_cmdline_vars($argv);
     dump_cmdline_vars();
 
-    $jar = "$ENV{JAVA_HOME}/bin/jar";
-    $javac = "$ENV{JAVA_HOME}/bin/javac";
-    $javah = "$ENV{JAVA_HOME}/bin/javah";
-    $javadoc = "$ENV{JAVA_HOME}/bin/javadoc";
+    our $jar = "$ENV{JAVA_HOME}/bin/jar";
+    our $javac = "$ENV{JAVA_HOME}/bin/javac";
+    our $javah = "$ENV{JAVA_HOME}/bin/javah";
+    our $javadoc = "$ENV{JAVA_HOME}/bin/javadoc";
+
+    our $dist_dir = abs_path($cmdline_vars{SOURCE_PREFIX});
 
-    $dist_dir = abs_path($cmdline_vars{SOURCE_PREFIX});
-    $jce_jar = $ENV{JCE_JAR};
+    our $class_dir = "$dist_dir/classes";
+    our $class_jar = "$dist_dir/$cmdline_vars{XPCLASS_JAR}";
 
-    $class_release_dir = $cmdline_vars{SOURCE_RELEASE_PREFIX};
-    $class_dir = "$dist_dir/classes";
-    $class_jar = "$dist_dir/$cmdline_vars{XPCLASS_JAR}";
+    our $class_release_dir = $cmdline_vars{SOURCE_RELEASE_PREFIX};
     $class_release_dir .= "/$cmdline_vars{SOURCE_RELEASE_CLASSES_DIR}";
+
+    our $javac_opt_flag = "-g";
     if( $ENV{BUILD_OPT} ) {
         $javac_opt_flag = "-O";
-    } else {
-        $javac_opt_flag = "-g";
     }
-    $jni_header_dir = "$dist_dir/private/jss/_jni";
 
+    our $jni_header_dir = "$dist_dir/private/jss/_jni";
+
+    our$jarFiles = "/usr/share/java/slf4j/slf4j-api.jar:/usr/share/java/commons-codec.jar";
```

Comment:
Do we care that there is no space between our and the var?

Reply:
Heh good catch. It looks like Perl doesn't care about spacing around scalars:

```perl
my$var="testing\n";
print$var;
```

prints:
```
testing
```

So go figure. I'll update this when I get time.